.travis.yml

```diff
@@ -51,7 +51,7 @@ script:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then export LD_LIBRARY_PATH=$LIBRARY_PATH ; fi
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then cargo build --features vulkan; else cargo build; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then cargo build --features metal; else cargo build; fi
-  - cargo test --all
+  - cargo test --all --features serialize
```

Comment:
Given that the feature only makes the types derive from `Serialize `/`Deserialize` and nothing more, it just has to compile correctly to pass. But if you really want, I can duplicate the line to re-run all tests without it.